ReplacingDocker/podman.sh

```diff
@@ -0,0 +1,287 @@
+    sudo ausearch -m path -ts recent -i | grep touch | tail -n 1 | grep --color=auto 'auid=[^ ]*'
+    echo ""
+
+    read -p "--> sudo docker run --privileged -v /:/host fedora touch /host/etc/shadow"
+    sudo docker run --privileged -v /:/host fedora touch /host/etc/shadow
+    echo ""
+
+    read -p "--> ausearch -m path -ts recent -i | grep touch | tail -n 1 | grep --color=auto 'auid=[^ ]*'"
+    sudo ausearch -m path -ts recent -i | grep touch | tail -n 1 |grep --color=auto 'auid=[^ ]*'
+    echo ""
+
+    read -p "--> clear"
+    clear
+}
+
+top() {
+    # Podman top commands
+    read -p "Podman top features"
+    echo ""
+
+    read -p "--> sudo podman run -d fedora sleep 1000"
+    sudo podman run -d fedora sleep 1000
+    echo ""
+
+    read -p "--> sudo podman top --latest pid hpid"
+    sudo podman top --latest pid hpid
+    echo ""
+
+    read -p "--> sudo podman top --latest label"
+    sudo podman top --latest label
+    echo ""
+
+    read -p "--> sudo podman top --latest seccomp"
+    sudo podman top --latest seccomp
+    echo ""
+
+    read -p "--> sudo podman top --latest capeff"
+    sudo podman top --latest capeff
+    echo ""
+
+    read -p "--> clear"
+    clear
+}
+
+pods() {
+    sudo podman kill -a
+    sudo podman rm -a -f
+    clear
+
+    read -p "--> sudo podman pod list"
+    sudo podman pod list
+    echo ""
+
+    read -p "--> sudo podman pod --name podtest create"
+    sudo podman pod create --name podtest
+    echo ""
+
+    read -p "--> sudo podman pod create --pod podtest -d fedora sleep 600"
+    sudo podman create --pod podtest -d fedora sleep 600
+    echo ""
+
+    read -p "--> sudo podman pod create --pod podtest -d fedora sleep 600"
+    sudo podman create --pod podtest -d fedora sleep 600
+    echo ""
+
+    read -p "--> sudo podman ps"
+    sudo podman ps
+    echo ""
+
+    read -p "--> sudo podman pod start podtest"
+    sudo podman pod start podtest
+    echo ""
+
+    read -p "--> sudo podman ps"
+    sudo podman ps
+    echo ""
+
+    read -p "--> sudo podman pod stop podtest"
+    sudo podman pod kill --signal KILL podtest
+    echo ""
+
+    read -p "--> sudo podman ps"
+    sudo podman ps
+    echo ""
+
+    read -p "--> sudo podman pod rm --force podtest"
+    sudo podman pod rm --force podtest
+    echo ""
+    clear
+
+    read -p "--> sudo podman pod list"
+    sudo podman pod list
+    echo ""
+}
+
+build() {
+    echo ""
+    # Already built an image with buildah installed in it
+    # and made buildah the entrypoint
+
+    sudo mkdir -p /var/lib/mycontainer
+    mkdir -p $PWD/myvol
+    cat >$PWD/myvol/Dockerfile <<_EOF
+FROM alpine
+ENV foo=bar
+LABEL colour=blue
+_EOF
+
+    read -p "--> cat Dockerfile"
+    cat $PWD/myvol/Dockerfile
+    echo ""
+    echo ""
+    read -p "--> sudo podman run -v \$PWD/myvol:/myvol:Z -v /var/lib/mycontainer:/var/lib/containers:Z buildah-ctr --storage-driver vfs bud -t myimage --isolation chroot /myvol"
```

Comment:
On a F28 VM with nothing installed except Podman, this failed and then the script went south after that.  If there's other required setup, we need to list that in the README.md

```
--> sudo podman run -v $PWD/myvol:/myvol:Z -v /var/lib/mycontainer:/var/lib/containers:Z buildah-ctr --storage-driver vfs bud -t myimage --isolation chroot /myvol
Trying to pull docker.io/buildah-ctr:latest...Failed
Trying to pull registry.fedoraproject.org/buildah-ctr:latest...Failed
Trying to pull quay.io/buildah-ctr:latest...Failed
Trying to pull registry.access.redhat.com/buildah-ctr:latest...Failed
Trying to pull registry.centos.org/buildah-ctr:latest...Failed
unable to pull buildah-ctr: 5 errors occurred:

* Error determining manifest MIME type for docker://buildah-ctr:latest: Error reading manifest latest in docker.io/library/buildah-ctr: errors:
denied: requested access to the resource is denied
unauthorized: authentication required

* Error determining manifest MIME type for docker://registry.fedoraproject.org/buildah-ctr:latest: Error reading manifest latest in registry.fedoraproject.org/buildah-ctr: manifest unknown: manifest unknown
* Error determining manifest MIME type for docker://quay.io/buildah-ctr:latest: Error reading manifest latest in quay.io/buildah-ctr: error parsing HTTP 404 response body: invalid character '<' looking for beginning of value: "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 3.2 Final//EN\">\n<title>404 Not Found</title>\n<h1>Not Found</h1>\n<p>The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.</p>\n"
* Error determining manifest MIME type for docker://registry.access.redhat.com/buildah-ctr:latest: Error reading manifest latest in registry.access.redhat.com/buildah-ctr: unknown: Not Found
* Error determining manifest MIME type for docker://registry.centos.org/buildah-ctr:latest: Error reading manifest latest in registry.centos.org/buildah-ctr: manifest unknown: manifest unknown
```